<launch>
    <!-- Publicadores de cámaras -->
    <node pkg="vc_battleship" type="usb_cam_publisher.py" name="usb_cam_publisher" output="screen" />
    <node pkg="vc_battleship" type="webcam_publisher.py" name="webcam_publisher" output="screen" />

    <!-- Nodo de tablero -->
    <node pkg="vc_battleship" type="board_node.py" name="board_node" output="screen">
        <param name="fps" value="30" />
        <param name="image_topic" value="/usb_cam/image_raw" />
    </node>

    <!-- Nodo de gestos -->
    <node pkg="vc_battleship" type="gesture_node.py" name="gesture_node" output="screen" />

    <!-- Nodo de lógica del juego -->
    <node pkg="vc_battleship" type="game_logic_node.py" name="game_logic_node" output="screen" />
</launch>
